.github/workflows/validate-renovate-config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           node-version: 18
 
       - name: Install Renovate
-        run: npm install -g renovate
+        run: npm install -g renovate@39.9.3
```

Comment:
i wonder whether renovate picks up on version numbers as part of `npm install xxxx` commands.
if it doesnt, we can move renovate to the devDependencies, where depdencies do get updated.

Reply:
npm doesn’t support hash-based pinning like GitHub Actions do. I tried few things but the node_module build itself propagates warning.
Here what i tried:
Create a package.json in .github/workflows directory.
Define renovate as a dependency with a specific version.
Commit the generated package-lock.json to lock the version and all sub-dependencies.
and in workflow add
```
- name: Install Renovate
  run: npm ci --prefix .github/workflows
```
  
![image](https://github.com/user-attachments/assets/c4933fc1-8bac-4d10-8d6c-bc542a9b063c)

Reply:
> we can move renovate to the devDependencies, where depdencies do get updated.
Going to try this now.